backend/app/tests/crud/test_token_blacklist_crud.py

```
@@ -83,62 +101,92 @@ def test_add_token_to_blacklist(MockAppModelsTokenBlacklist: MagicMock, mock_db_
     mock_query_obj.where.assert_called_once()
 
 
 @patch("app.crud.select")
-@patch("app.models.TokenBlacklist")
-def test_check_token_in_blacklist_not_found(MockAppModelsTB:MagicMock, mock_select_crud: MagicMock, mock_db_session: MagicMock, sample_token_data: dict):
+@patch("app.crud.TokenBlacklist")
+def test_check_token_in_blacklist_not_found(
+    MockAppModelsTB: MagicMock,
+    mock_select_crud: MagicMock,
+    mock_db_session: MagicMock,
+    sample_token_data: dict,
```

Ruff (0.11.9)

Unused function argument: `sample_token_data`

(ARG001)